validation/src/test/scala/hmda/validation/rules/lar/macro/SimplifiedMacroSpec.scala

```diff
@@ -0,0 +1,37 @@
+package hmda.validation.rules.lar.`macro`
+
+import com.typesafe.config.ConfigFactory
+import hmda.model.fi.lar.LoanApplicationRegister
+
+trait SimplifiedMacroSpec extends MacroSpec {
+
+  def multiplier: Double
+
+  def relevantLar(lar: LoanApplicationRegister): LoanApplicationRegister
+  def irrelevantLar(lar: LoanApplicationRegister): LoanApplicationRegister
+
+  protected def simplifiedPropertyTests(name: String, multiplier: Double, relevantLar: LoanApplicationRegister => LoanApplicationRegister, irrelevantLar: LoanApplicationRegister => LoanApplicationRegister): Unit = {
```

Comment:
This is definitely a great way to trim down on duplicated code!  One thing I would suggest would be to rename this... Something like `lessThanOrEqualToPropertyTests`.  Just to give some idea of what we're expecting from this function, since it's specific to that behavior.